src/app/_services/auth.service.ts

```diff
@@ -157,8 +157,9 @@ export class AuthService {
       //  So, create a reference to it
       window.myLoadMashup = bootstrapShell.loadMashup;
 
-      // For experimentation, save a reference to loadPortal, too!
+      // For experimentation, save a reference to loadPortal,loadDefaultPortal too!
```

Comment:
We can eliminate the "For experimentation, " as no longer an experiment but we are actually using this approach